src/styles/global.scss

```diff
@@ -13,8 +13,7 @@
 @import './variables';
 
 ::-webkit-scrollbar {
-  width: 1px;
-  background-color: transparent;
+  display: none;
```

Comment:
Isn't this conflicting with #776 ?

Reply:
this is the same change, git should handle this well

Reply:
I needed to do the same change as part of the fix. My intention is to rebase and remove the duplicated commit once #776 is merged. 

I've also added it as depending to that PR in the description. Please let me know if there's a more appropriate way to handle this case.